components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -310,7 +310,10 @@ - (void)commonMDCFlexibleHeaderViewInit {
       (UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight);
   [super addSubview:_contentView];
 
-  self.backgroundColor = [UIColor lightGrayColor];
+  if (![MDCFlexibleHeaderView appearance].backgroundColor) {
```

Comment:
I think this is a larger question.
I'm going to +1 this PR and we can determine the 'standard' way to handle this for all of our components.